### DISCUSSION TITLE
Request For Starship EVO

### DISCUSSION THREAD

**Discussion by Arutus** [tsunamayo/Starship-EVO#4105] [Suggestions]
Hello Tsuna if possible can we get a zoom control function for third person flight because as of right now while flying large ships the camera zooms out to much during movement and boosting, And also if possible a speed slider for rail blocks would be great for train/tram systems in large bases or ships.

(to reiterate I do not mean the look around zoom which is controllable, I mean the fight mode in which the camera is locked behind the ship in third person, I would like you to be able to set the zoom distance in that mode.)

**Comment by Jorpis-Jorah** [comment]
I think a zoom feature would be good in general, like star citizen

**Reply by tsunamayo** [reply]
@Jorpis-Jorah I havent played it, if you could tell me more?

**Reply by Jorpis-Jorah** [reply]
In sc, if you press a certain button, it will zoom in and decrease your fov, which yes, would be a nice thing to have when looking at distant objects.

**Comment by tsunamayo** [comment]
@Arutus Okay, so would a view distance edit in the spaceship core be enough for your need? Or does it have to be real-time? Thanks

**Reply by Arutus** [reply]
real time while your flying
so you can control your visibility

**Reply by Arutus** [reply]
also just to add on to what I mentioned please have this be a scrollable distance with the middle mouse wheel that lets us zoom in and out in flight control mode. without the zoom out during boosting or moving

**Reply by tsunamayo** [reply]
I dont want to use middle mouse wheel for that, I might need it for throttle control.

**Reply by tsunamayo** [reply]
I just dont get the need to control this all the time. If it is for large ship then why a simple tweak in the spaceship core is not enough?

**Reply by Arutus** [reply]
because a constant control over camera is integral to ones view over whats happening while the ship is in motion or in combat.
Throttle could be controlled with the W/S buttons like it is now.

**Reply by Arutus** [reply]
Imagine being unable to quickly increase scale in third person because you are taking battle damage to a sector of your large ship and cant see it because its to zoomed out. Plus its kind of a control scheme any space flight building game has with third person much like space engineers, its just a better idea to have it assigned to a mouse button for constant control as a player is gonna want to zoom in and out of there ship in 3rd person for a variety of reasons
